mne/decoding/csp.py

```diff
@@ -31,12 +31,13 @@ class CSP(TransformerMixin, BaseEstimator):
     n_components : int, defaults to 4
         The number of components to decompose M/EEG signals.
         This number should be set by cross-validation.
-    reg : float | str | None, defaults to None
-        if not None, allow regularization for covariance estimation
-        if float, shrinkage covariance is used (0 <= shrinkage <= 1).
-        if str, optimal shrinkage using Ledoit-Wolf Shrinkage ('ledoit_wolf')
-        or Oracle Approximating Shrinkage ('oas').
-    log : None | bool, defaults to None
+    reg : float | str | None (default None)
+        If not None (same as ``'empirical'``, default), allow
+        regularization for covariance estimation.
+        If float, shrinkage is used (0 <= shrinkage <= 1).
+        For str options, ``reg`` will be passed to ``method`` to
+        :func:`mne.compute_covariance`.
```

Comment:
there is no mention anymore to 'ledoit_wolf' or 'oas'

Reply:
Backward compatibility. The former is in `compute_covariance` and the latter in `XDawn` / `CSP` in 0.15

Reply:
Oh I misread that. The idea is that the user should look at the doc of `compute_covariance`, I'll make it clearer